src/shared/platforms/linux/sources.cmake

```diff
@@ -0,0 +1,11 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+find_package(PkgConfig REQUIRED)
+pkg_check_modules(polkit REQUIRED IMPORTED_TARGET polkit-gobject-1)
```

Comment:
`libsecret-1` is very much a shared dependency in order to realize the cryptosettings on Linux, but `polkit-gobject-1` is only required for the VPN module in order to communicate on the system D-Bus.